### ISSUE TITLE
How to get extension attributes details in the webhook

### ISSUE THREAD

**Issue by zubinudwadiabts** [mageplaza/magento-2-webhook#66]
we need the details for the payment_additional_info which is being generated from stripe. It shows up in the order details under extension_attributes. But how do we get the data in the webhook body

The details which we want
![orderdemo](https://user-images.githubusercontent.com/48488724/114415876-b740a480-9bcd-11eb-871e-01b7af916313.JPG)

**Comment by phutx** [comment]
Hi @zubinudwadiabts ,
I'm Onesh from Mageplaza Support Team.
Please help me download the zip file, unzip it, and replace the file to the path below:
- app/code/Mageplaza/Webhook/Helper/Data.php

Via composer:
- mageplaza/module-webhook/Helper/Data.php

After that, kindly use the following body:

"Key":{{ item.key}}
"Value":{{ item.value }}

[Data.zip](https://github.com/mageplaza/magento-2-webhook/files/6308080/Data.zip)

Thanks!

**Comment by zubinudwadiabts** [comment]
I replaced the file and still didn't get any data

![webhook_1](https://user-images.githubusercontent.com/48488724/114715892-27c3fe80-9d51-11eb-9bec-a5bb65b0a2ef.JPG)
![webhook_3](https://user-images.githubusercontent.com/48488724/114716093-580b9d00-9d51-11eb-967a-2db8dd017188.JPG)

**Comment by phutx** [comment]
Hi @zubinudwadiabts,
Please help me replace the data.zip file and create a new order. After that, kindly access and capture the var/log/test.log file for me.
[Data.zip](https://github.com/mageplaza/magento-2-webhook/files/6323074/Data.zip)
Thanks!

**Comment by zubinudwadiabts** [comment]
The test.log file is empty

[test.zip](https://github.com/mageplaza/magento-2-webhook/files/6323664/test.zip)

**Comment by zubinudwadiabts** [comment]
The test.log file was generated, but it's empty.

[test.zip](https://github.com/mageplaza/magento-2-webhook/files/6323832/test.zip)

**Comment by phutx** [comment]
Hi @zubinudwadiabts,

Please try again with this file.
[Data.zip](https://github.com/mageplaza/magento-2-webhook/files/6323870/Data.zip)
Thanks.

**Comment by zubinudwadiabts** [comment]
still empty 😞

**Comment by phutx** [comment]
Hi @zubinudwadiabts,
Have you created a hook in the managet hook yet?
https://prnt.sc/11oao9x

**Comment by zubinudwadiabts** [comment]
yes, and it's even working, because we tried sending the order id to my API and it went through

**Comment by zubinudwadiabts** [comment]
![order webhook](https://user-images.githubusercontent.com/48488724/115241049-02af0180-a13e-11eb-93a5-a5fa56b79fde.JPG)

**Comment by phutx** [comment]
Hi @zubinudwadiabts,

Thank you a lot for keeping me updated on this news.

I regret that my solution is ineffective in this case. For further support, please submit a ticket with your site's information here:
https://www.mageplaza.com/contact.html
So I can investigate more details and reach the best solution to your current request.

I do value your help in this matter and look forward to hearing from you soon.

**Comment by zubinudwadiabts** [comment]
hey @

> Hi @zubinudwadiabts,
> 
> Thank you a lot for keeping me updated on this news.
> 
> I regret that my solution is ineffective in this case. For further support, please submit a ticket with your site's information here:
> https://www.mageplaza.com/contact.html
> So I can investigate more details and reach the best solution to your current request.
> 
> I do value your help in this matter and look forward to hearing from you soon.

I have created a ticket with all the information needed 
ticket id #112686
